JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R

```diff
@@ -238,26 +252,54 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
 
 
 .BayesianReliabilityCheckLoadings <- function(dataset, variables) {
-
-	prin <- psych::principal(dataset)
-	idx <- prin[["loadings"]] < 0
-	sidx <- sum(idx)
-	if (sidx == 0) {
-	  return()
-	} else {
-	  hasSchar <- if (sidx == 1L) "" else "s"
-	  footnote <- gettextf("The following item%s correlated negatively with the scale: %s. ",
-	                       hasSchar, paste0(variables[idx], collapse = ", "))
-	  return(footnote)
-	}
-
+  if (ncol(dataset > 2)) {
+    prin <- psych::principal(dataset)
+    idx <- prin[["loadings"]] < 0
+    sidx <- sum(idx)
+    if (sidx == 0) {
+      return("")
+    } else {
+      hasSchar <- if (sidx == 1L) "" else "s"
+      footnote <- gettextf("The following item%s correlated negatively with the scale: %s. ",
+                           hasSchar, paste0(variables[idx], collapse = ", "))
+      return(footnote)
+    }
+  } else {
+    return("")  
+    }
 }
 
 
 # ----------------------------- tables ------------------------------------
 .BayesianReliabilityScaleTable <- function(jaspResults, model, options) {
   
-  if (!is.null(jaspResults[["scaleTable"]])) {
```

Comment:
Well, this causes an error for me, because when no coefficients are checked an empty ScaleTable is displayed, when I then check omega as a coefficient, the Table stays empty. Do you have an idea how to resolve this?

Reply:
Ah I found it, the dependencies for the empty table were incomplete.